docs/use_cases/retrieval_augmented_generation_eval.md

```diff
@@ -0,0 +1,181 @@
+<!-- SEO: Retrieval augmented generation Evaluation - TODO Summary
+ -->
+
+# Evaluating Retrieval Augmented Generation
+
+
+## Understanding Retrieval-Augmented Generation
+
+<img src=/assets/use_cases/retrieval_augmented_generation_eval/rag_qdrant.jpg alt="Implementation of RAG using Qdrant as a vector database" data-size="100" />
+
+**Disclaimer**:
+The provided implementation serves as an example using  [Qdrant](https://qdrant.tech). Alternative Vector Databases are also available. To determine the most suitable Vector Database for your specific use case, please refer to the [Vector DB feature matrix](https://vdbs.superlinked.com/)
+
+**RAG** stands for **Retrieval Augmented Generation**, and it probably is the most useful application of large language models lately.
+It is the technique that combines the strengths of both retrieval and generation models. The retrieval is usually based on dense vector search, in combination with a text generation model like GPT.
+
+RAG has received significant attention due to its ability to enhance content generation by leveraging existing information effectively. 
+Its capacity to amalgamate specific, relevant details from multiple sources and generate accurate and relevant content has a lot of potential in various domains like content creation, question & answer application, and information synthesis.
+
+Read more: [here](https://hub.superlinked.com/retrieval-augmented-generation)
+
+## Why do we need RAG?
+
+RAG plays a crucial role to significantly enhance [vector search](https://qdrant.tech/documentation/overview/vector-search/) with the power of Large Language Model (LLM), enabling dynamic content generation based on the retrieved knowledge.
+
+It proves invaluable for tasks requiring detailed, coherent explanations, summaries, or responses that transcend the explicit data stored in vectors. 
+While vector search efficiently finds similar items, RAG takes a step further, offering content synthesis and a deeper level of understanding.
```

Comment:
I would also highlight that compared to just retrieving chunks and presenting them to the user, every answer can be unique and tailored to the exact query. Maybe it is there implicitly but I would I think state that explicitly.

Reply:
Done! Please review